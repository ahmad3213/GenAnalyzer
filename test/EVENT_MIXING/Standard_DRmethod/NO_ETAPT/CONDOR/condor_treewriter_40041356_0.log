000 (40041356.000.000) 11/08 09:33:59 Job submitted from host: <131.225.188.57:9618?addrs=131.225.188.57-9618&noUDP&sock=2418_2017_3>
...
001 (40041356.000.000) 11/08 09:34:27 Job executing on host: <131.225.205.170:9618?addrs=131.225.205.170-9618&noUDP&sock=1456_45fb_3>
...
006 (40041356.000.000) 11/08 09:34:35 Image size of job updated: 68
	68  -  MemoryUsage of job (MB)
	69932  -  ResidentSetSize of job (KB)
...
005 (40041356.000.000) 11/08 09:36:28 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:00:38, Sys 0 00:00:23  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:38, Sys 0 00:00:23  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	727  -  Run Bytes Sent By Job
	16380  -  Run Bytes Received By Job
	727  -  Total Bytes Sent By Job
	16380  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :        0        1         1 
	   Disk (KB)            :       42        7   1648013 
	   Memory (MB)          :        6     2048      2048 
...
