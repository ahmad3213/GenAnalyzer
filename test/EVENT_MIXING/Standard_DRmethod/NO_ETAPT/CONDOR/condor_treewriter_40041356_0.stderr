In file included from input_line_10:1:
/storage/local/data1/condor/execute/dir_42439/DoubleJpsi_mix_noetapt_0.cxx:20:10: fatal error: 'scriptName1' file not found
#include "scriptName1"
         ^~~~~~~~~~~~~
